lib/elementWrapperUtils.js

```diff
@@ -122,6 +126,15 @@ const generateElementWrapper = function(
       if (!elems[0]) throw new Error(`${this.description} not found`);
       return await elems[0].text();
     },
+    isVisible: async function(retryInterval, retryTimeout) {
+      const elems = await this.elements();
+      if (elems.length < 1) {
+        descEvent.emit('success', 'Element not Visible');
```

Comment:
I think it's better to throw an error if no elements are found.

Reply:
I want to keep this consistent with `exists`. What u think?